### PR TITLE
Fix doctest

### PR DESCRIPTION
**Pull request by okuta**
- Use `cp` and `np`.
- Use `doctest` instead of `testcode`.

### CODE REVIEW

File: docs/source/tutorial/basic.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ The :class:`cupy.ndarray` class is in its core, which is a compatible GPU altern
```
Comment (delta2323) [inline docs/source/tutorial/basic.rst]: We need the description like "In the following code, `cp` is an abbreviation of `cupy`, as `np` is `numpy` as with customarily done.

Comment (okuta) [inline docs/source/tutorial/basic.rst]: Thank you! I will add.

**Comment by okuta** [comment]
I fixed @delta2323

File: docs/source/tutorial/basic.rst (hunk 1 above)
Comment (delta2323) [inline docs/source/tutorial/basic.rst]: Sorry, I made a mistake. Please change to "as *is* customarily done".

**Comment by delta2323** [comment]
LGTM

**Pull request merged**